Add Documentation for metrics

jcscottiii opened the pull request.

Commit:
Add Documentation for metrics

jessieay commented on content/ops/metrics.md:
```diff
@@ -0,0 +1,31 @@
+1. Please indicate if you need an `admin` account or `regular` account.
+
+# Updating the Metrics containers.
+1. SSH into jumpbox
```
instructions or link on how to do this?

jcscottiii replied:
Done :)

jessieay commented on content/ops/metrics.md:
```diff
@@ -0,0 +1,31 @@
+1. Currently, the dashboards are looking for the `Name` to be `cf_prd`, `cf_np`, `bosh_prd`, or `bosh_np`. So if you are updating one of those, please ensure the name fits exactly.
+1. Set the `Type` to `InfluxDB 0.9.x`
+1. Given the AWS Internal IP of the Docker Host (Please refer to the AWS EC2 Console), Set the `Url` to `http://<internal-ip>:8086`
+1. Set the `Database`, `User`, and `Password` as appropriate.
```
instructions or link on how to do this? (probably obvious to some, but not to me)

jcscottiii replied:
Done :)

Commit:
Adding more details to the metrics doc

This commit adds clarifying steps to the different tasks laid out in
the metrics doc either by adding more details or adding links to docs.
